tests/DCPS/RtpsRelay/Smoke/run_stress_test1.py

```diff
@@ -0,0 +1,514 @@
+                                 '-RestartDetection', '1',
+                                 '-LogThreadStatus', '0',
+                                 '-AdmissionControlQueueSize', '10',
+                                 '-AdmissionControlQueueDuration', '5',
+                                 '-AdmissionMaxParticipantsRange', '9500-10000',
+                                 '-LogUtilizationChanges', '1',
+                                 '-ORBLogFile', log_path,
+                                 )
+
+class MonitorSched(OneshotSched):
+    def __init__(self, **params):
+        OneshotSched.__init__(self, **params)
+
+    def callback(self):
+        log_path = f'monitor.log'
+        if os.path.exists(log_path):
+            os.remove(log_path)
+        return self.proc_man.run('monitor', f'{DDS_ROOT}/tests/DCPS/RtpsRelay/Smoke/monitor',
+                                 '-DCPSConfigFile', 'monitor.ini',
+                                 '-DCPSDebugLevel', '1',
+                                 '-ORBVerboseLogging', '1',
+                                 '-DCPSTransportDebugLevel', '1',
+                                 '-ORBLogFile', log_path,
+                                 '-DCPSPendingTimeout', '3',
+                                 )
+
+def run_sim(*scheds):
+    event_loop = EventLoop()
+    proc_man = ProcMan()
+    for sched in scheds:
+        sched.ready(event_loop, proc_man)
+    event_loop.run_events()
+
+
+if __name__ == '__main__':
+    argp = argparse.ArgumentParser()
+    argp.add_argument('count', type=int,
+        help='Fixed count of client pairs. If using --ramp-up, ' +
+          'then this is the max count with 0 meaning unlimited.')
+    argp.add_argument('--max-time', type=int, default=0)
+    argp.add_argument('-r', '--ramp-up', type=int, default=0,
+        help='Gradually increase load in these increments')
+    argp.add_argument('--dummy', action='store_true')
+    argp.add_argument('--profile', action='store_true', help='Profile the relay with massif')
+    args = argp.parse_args()
+
+    # Generate keys, certificates, and permissions documents
+    os.system(f'$DDS_ROOT/tools/scripts/dpm.py --init')
+    os.system(f'$DDS_ROOT/tools/scripts/dpm.py --sign governance.xml governance.xml.p7s')
+
+    for i in range(1,3):
+        relay_name = f'relay_{i}'
+        relay_sn = f'CN={relay_name}'
+        os.system(f'$DDS_ROOT/tools/scripts/dpm.py --gencert {relay_name} "/{relay_sn}"')
+        permissions = f'''<?xml version="1.0" encoding="utf-8"?>
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.omg.org/spec/DDS-SECURITY/20170901/omg_shared_ca_permissions.xsd">
+  <permissions>
+    <grant name="ShapesPermissionTOC">
+      <subject_name>{relay_sn}</subject_name>
+      <validity>
+        <!-- Format is CCYY-MM-DDThh:mm:ss[Z|(+|-)hh:mm] in GMT -->
+        <not_before>2015-09-15T01:00:00</not_before>
+        <not_after>2025-09-15T01:00:00</not_after>
```

Comment:
Should we go ahead and update timestamps now?  5 months doesn't seem like long enough -- but we could always come back to it when the tests start failing.